terminal/src/lib.rs

```diff
@@ -202,6 +205,20 @@ pub mod components {
         println!("{}", style(text).dim());
     }
 
+    pub fn markdown(content: &str) {
+        PrettyPrinter::new()
+            .language("md")
+            .grid(true)
+            .header(true)
+            .input(
+                BatInput::from_bytes(content.as_bytes())
+                    .name("tmp.md")
+                    .kind("File"),
+            )
+            .print()
+            .unwrap();
```

Comment:
Since `bat` brings in lots of dependencies, I think it would be better to use the system `bat`, ie. check if the `bat` command exists, and if so, use it, otherwise print without highlighting.

Reply:
Ok, switched to shelling out to `bat` (if available.)